make/common/native/Paths.gmk

```diff
@@ -200,6 +191,17 @@ define SetupOutputFiles
   endif
 endef
 
+################################################################################
+define RemoveSuperfluousOutputFiles
```

Comment:
I left out the third category of macros that are neither Setup nor Create, since they can do all kind of strange stuff. But now I added a comment about it; hope you feel it's enough.